### ISSUE TITLE
Gitlab windows64 job dying "There is not enough space on the disk."

### ISSUE THREAD

**Issue by SkySkimmer** [coq/coq#9510]
eg https://gitlab.com/coq/coq/-/jobs/157955159 https://gitlab.com/coq/coq/-/jobs/157950775
cc @MSoegtropIMC

**Comment by MSoegtropIMC** [comment]
@SkySkimmer : I was a bit sloppy with the runner maintenance cause of the Azure move. Will do it now.

**Comment by MSoegtropIMC** [comment]
I cleaned up all servers.

There was on average about 1 job which failed to cleanup per month, but interestingly most failures are on just a few dates. Most common was January 24th and February 6th. I guess this happens when somehow the connection between the runner and gitlab breaks - the runner seems to abort the script then which leads to the cleanup failure.

I killed one recent job (from today after 5pm) which looked bogus.

**Comment by Zimmi48** [comment]
@MSoegtropIMC It seems that all Windows runners are now offline. Is there something else that needed to be done?

**Comment by MSoegtropIMC** [comment]
Hmm strange I saw jobs appearing while I was doing the cleanup.

Actually I don't know that much how the runner setup is done, I did only take care of the problem that sometimes runner jobs are dying in the middle of the execution without any reason and don't do their cleanup which then had a bad impact on other jobs. The solution was to use uniquely named folders and clean up debris from time to time (once per month is usually enough).

I can look into this, but it might take a while.

Let me look at some failing jobs ...
